library/menu/src/PatientMenuRole.php

```diff
@@ -181,14 +181,16 @@ public function displayHorizNavBarMenu()
                 // flatten to only show children items
                 foreach ($value->children as $children_key => $children_value) {
                     $link = ($children_value->pid != "true") ? $children_value->url : $children_value->url . attr($pid);
-                    $list = '<li class="oe-bold-black" id="nav-list'. $li_id.'">';
+                    $class = isset($children_value->class) ? $children_value->class : '';
+                    $list = '<li class="oe-bold-black ' . $class . '" id="' . $children_value->menu_id . '">';
                     $list .= '<a href="' . $link . '" onclick="' . $children_value->on_click .'"> ' . text($children_value->label) . ' </a>';
                     $list .= '</li>';
                 }
                 echo $list. "\r\n";
             } else {
                 $link = ($value->pid != "true") ? $value->url : $value->url . attr($pid);
-                $list = '<li class="oe-bold-black"id="nav-list'. $li_id.'">';
+                $class = isset($value->class) ? $value->class : '';
+                $list = '<li class="oe-bold-black ' . $class . '" id="' . $value->menu_id . '">';
```

Comment:
should place attr() around both the above variables

Reply:
Done